creator/data_templates/factories.py

```diff
@@ -0,0 +1,56 @@
+import pytz
+import factory
```

Comment:
```suggestion
import factory
import factory.fuzzy
```

Without this line you get `AttributeError: module 'factory' has no attribute 'fuzzy'` when trying to use this file